### ISSUE TITLE
In single selection mode, aria label is not announced after the single selection is made

### ISSUE THREAD

**Issue by radhikakhetan** [OfficeDev/office-ui-fabric-react#11695]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

In the picker we pass an aria label for accessibility for the field. When no suggestion is selected, this aria label is announced. But when some suggestion is selected, this aria-label is not announced. This is for single selection mode. Is this by design or an existing issue ?

#### Actual behavior:

Aria label not announced after single selection

#### Expected behavior:

Aria label should be announced along with selection.

**Comment by xugao** [comment]
@radhikakhetan - 
1. i could not repro this issue. could you provide a codepen contains repro? I checked the dom the aria-label is correctly labelled in both cases of suggestion being selected/unselected.
2. details steps leading to aria label not being read, including which browser/screen reader you are using.

**Comment by radhikakhetan** [comment]
Hi @xugao. 
1. Have you tried for the case when a user can only select on person ? It reproes only for the picker on which a single value can be selected.
2. It reproes for all the browsers.

**Comment by xugao** [comment]
@radhikakhetan - did you try latest version of fabric? which type of screen reader (e.g. Narrator, NVDA, etc.) you repro with?  i tried in our example page here : https://fabricweb.z5.web.core.windows.net/oufr/7.82.2/#/examples/peoplepicker

**Comment by radhikakhetan** [comment]
@xugao I tried in with version 7.73.0 and it reproes with Narrator with MacOS, Narrator both. The example you are trying with is incorrect. You need to try with example where only a single persona can be selected i.e. uniselction mode. Essentially by adding attribute itemLimit as 1. Please try with this.

**Comment by xugao** [comment]
@radhikakhetan - pls provide me a codepen to demo the component which has a repro.
